gradle-conjure/src/test/groovy/com/palantir/gradle/conjure/TestVersions.java

```diff
@@ -19,7 +19,7 @@
 public final class TestVersions {
     private TestVersions() {}
 
-    public static final String CONJURE = "4.11.2";
+    public static final String CONJURE = "4.30.0";
```

Comment:
Updated so that the `--verbose` flag is actually used rather than listed as an unmatched input -- in practice it's not necessary for the test to pass, and we're only validating plumbing. I'll revert this out if it causes anything to fail.